config/locales/en.json

```diff
@@ -188,11 +188,12 @@
 	"conditions.save_condition": "Save condition",
 	"conditions.symptoms_began": "Symptoms began: ",
 	"conditions.title": "Medical conditions",
-	"confirmation.intro": "Confirmation Intro goes here",
-	"confirmation.save_data": "Save data",
-	"confirmation.thank_you": "Thank you!",
-	"confirmation.thank_you_participating": "Thank you for participating!",
-	"confirmation.title": "Confirmation Title",
+	"confirmation.title": "You have submitted %s's medical report",
+	"confirmation.you_have_submitted": "You have submitted a medical report to support %s's application for Canada Pension Plan Disability Benefits",
+	"confirmation.what_happens_next": "What happens next",
+	"confirmation.what_happens_next_1": "Service Canada will begin assessing %s's application for Canada Pension Plan Disability Benefits. Service Canada might contact you in the future if they need any further information about %s's health or work capacity.",
```

Comment:
The second `%s` is rendering as 'undefined' for some reason.

Reply:
That's because I forgot to pass the name in twice! (done)